### ISSUE TITLE
Do not support linking to existing BDV/XML images

### ISSUE THREAD

**Issue by tischi** [mobie/mobie-viewer-fiji#1115]
@constantinpape @martinschorb @jhennies @K-Meech 

Currently we have in the JAVA API:

```
 public void addBdvFormatImage ( SpimData spimData, String imageName, String datasetName,
									ProjectCreator.ImageType imageType, ProjectCreator.AddMethod addMethod, String uiSelectionGroup, ImageDataFormat imageDataFormat, boolean exclusive ) throws SpimDataException, IOException {

```

This is for "adding/linking" to already existing BDV/XML images. 

To simplify the code and it's maintenance I would like to remove this and only support linking to already existing OME-Zarr.

OK?

**Comment by constantinpape** [comment]
I think that's fine.

**Comment by K-Meech** [comment]
Hi @tischi - do you mean you only want to support moving/copying/linking to ome-zarr and remove all support for n5? Do you also want to remove n5 support for adding images currently open in ImageJ?

**Comment by tischi** [comment]
@K-Meech Yes.

**Comment by K-Meech** [comment]
@tischi - sure, that makes sense! My only concern would be that people who have existing n5 projects would no longer be able to edit / add to them via the project creator. So it would be good to give people some warning that this will be removed e.g. getting in touch with the main users  / making a post on image.sc etc ?

**Comment by tischi** [comment]
@K-Meech good point, do you know such people? Could you write such a forum post and ping them?

**Comment by martinschorb** [comment]
It would probably affect 97% of the volume-EM projects

**Comment by K-Meech** [comment]
@tischi I'm not sure who is using this at the moment, beyond the volume EM folks as @martinschorb said. You could consider adding a deprecation warning for a few months that gets printed when the project creator is opened?

**Comment by martinschorb** [comment]
Janelia runs N5 and historically we mostly created N5 as well.
But your changes would only affect the creator?
If so and the restriction is to no longer being able to add new sources, I guess that's fine. Python would still allow it...

**Comment by martinschorb** [comment]
Then from my side nothing speaks against going ahead with the cleanup.
